src/v2/Utils/setupTestWrapper.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react"
+import { mount } from "enzyme"
+import { QueryRenderer } from "react-relay"
+import { MockPayloadGenerator, createMockEnvironment } from "relay-test-utils"
+import { GraphQLTaggedNode, OperationType } from "relay-runtime"
+
+type SetupTestWrapper<T extends OperationType> = {
+  Component: ((props: T["response"]) => JSX.Element) | React.ComponentType
+  query: GraphQLTaggedNode
+  variables?: T["variables"]
+}
+
+export const setupTestWrapper = <T extends OperationType>({
+  Component,
+  query,
+  variables = {},
+}: SetupTestWrapper<T>) => {
+  const getWrapper = (mocks = {}) => {
+    const env = createMockEnvironment()
+
+    const TestRenderer = () => (
+      <QueryRenderer<T>
+        environment={env}
+        variables={variables}
+        query={query}
+        render={({ props, error }) => {
+          if (props) {
+            return <Component {...props} />
+          } else if (error) {
+            console.error(error)
+          }
+        }}
+      />
+    )
+
+    const wrapper = mount(<TestRenderer />)
+
+    env.mock.resolveMostRecentOperation(operation => {
+      return MockPayloadGenerator.generate(operation, { ...mocks })
+    })
+
+    wrapper.update()
+
+    return wrapper
+  }
+
+  return getWrapper
```

Comment:
It could be beneficial for intent and refactoring if this returned an object:

```
return { getWrapper }
```

Then devs would know that it needs to be invoked.